CHANGELOG.md

```diff
@@ -1,18 +1,17 @@
 # Release History
 
-## 0.0.15 (Unreleased)
+## 0.0.15 (2025-05-09)
 
 ### Features Added
 
 - Support for getting properties and runtime information for Azure Service Bus queues, topics, and subscriptions. https://github.com/Azure/azure-mcp/pull/150/
 - Support for peeking at Azure Service Bus messages from queues or subscriptions. https://github.com/Azure/azure-mcp/pull/144
-
-### Breaking Changes
-
-### Bugs Fixed
+- Adds Best Practices tool that provides guidance to LLMs for effective code generation. https://github.com/Azure/azure-mcp/pull/134
 
 ### Other Changes
 
+- Disabled Parallel testing for Live Tests https://github.com/Azure/azure-mcp/pull/151
```

Comment:
We only disable parallel testing in the pipeline.

Reply:
Made the clarification :)